tests/plots/test_decision.py

```diff
@@ -15,7 +17,108 @@ def test_random_decision(random_seed):
         rs.standard_normal(size=(20, 5)),
         show=False
     )
-    pl.close()
+    plt.close()
```

Comment:
Nitpick but I don't we need to bother closing the plots, as that's done by a fixture in `tests/plots/contest.py`

Reply:
done